pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -30,6 +31,10 @@ import (
 	multierror "github.com/hashicorp/go-multierror"
 	"github.com/prometheus/client_golang/prometheus"
 
+	"bytes"
```

Comment:
nit: run goimports, these should be up with the first group of golang std lib imports.

Reply:
I ran goimports twice.. and even ran make fmt.. this doesn't seem to move.

Reply:
Can you fix manually then please - either way it shouldn't be merged with imports out of place.